pkg/adaptor/rethinkdb.go

```diff
@@ -31,10 +33,24 @@ type Rethinkdb struct {
 	client *gorethink.Session
 }
 
+// rethinkDbConfig provides custom configuration options for the RethinkDB adapter
+type rethinkDbConfig struct {
```

Comment:
This needs to be publicly accessible so that it can be properly "Registered" [here](https://github.com/compose/transporter/blob/master/pkg/adaptor/registry.go#L21) as:

```go
Register("rethinkdb", "a rethinkdb sink adaptor", NewRethinkdb, RethinkdbConfig{})
```